frontend/src/containers/Ledger.js

```diff
@@ -221,16 +227,15 @@ export function rejectExp(expenseId) {
  */
 export function payExp(expenseId) {
   const {
-    collective,
     fetchPendingExpenses,
     fetchTransactions,
     payExpense,
     notify
   } = this.props;
 
-  return payExpense(collective.id, expenseId)
-    .then(() => fetchPendingExpenses(collective.slug))
-    .then(() => fetchTransactions(collective.slug))
+  return payExpense(this.collective.id, expenseId)
+    .then(() => fetchPendingExpenses(this.collective.slug))
```

Comment:
Could you add a `notify` here as well? Even better if references the expense data, so you know exactly which one and what was paid. "Expense paid: $61 to Jake Ming (jake@ming.com)" :)